### PR TITLE
added user page, and scroll to top

### PR DESCRIPTION
**Pull request by Ashutosh741**

### CODE REVIEW

File: src/component/questions/QuestionSeries.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      options: [
+        "Central Style Sheets",
+        "Cascading Style Sheets",
+        "Cascading Simple Sheets",
+        "Cars SUVs Sailboats",
+      ],
+      correctAnswerIndex: 1,
+    },
+    {
+      question: "Which language runs in a web browser?",
```
Comment (thakursachin05) [inline src/component/questions/QuestionSeries.jsx:33]: wrong question model

File: src/component/questions/Question.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        Accordion Item #2
+      </button>
+    </h2>
+    <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#accordionExample">
+      <div class="accordion-body">
+        <strong>This is the second item's accordion body.</strong> It is hidden by default, until the collapse plugin adds the appropriate classes that we use to style each element. These classes control the overall appearance, as well as the showing and hiding via CSS transitions. You can modify any of this with custom CSS or overriding our default variables. It's also worth noting that just about any HTML can go within the <code>.accordion-body</code>, though the transition does limit overflow.
+      </div>
+    </div>
+  </div>
+            </div>
```
Comment (thakursachin05) [inline src/component/questions/Question.jsx:66]: no hard coded explanation, or if this is not created by you then please we can approve

File: src/pages/user/UserProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+                      </p>
+                    </div>
+                  </div>
+                  <hr class="my-4" />
+                  <ul class="list-group list-group-flush">
+                    <li class="list-group-item d-flex justify-content-between align-items-center flex-wrap">
+                      <h6 class="mb-0">
+                        <svg
+                          xmlns="http://www.w3.org/2000/svg"
+                          width="24"
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:54]: no hard coded , create one user model right ? with required fields

File: src/pages/user/UserProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+                          height="24"
+                          viewBox="0 0 24 24"
+                          fill="none"
+                          stroke="currentColor"
+                          stroke-width="2"
+                          stroke-linecap="round"
+                          stroke-linejoin="round"
+                          class="feather feather-github me-2 icon-inline"
+                        >
+                          <path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"></path>
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:86]: why ? xmls is requried atleast clean the code

File: src/pages/user/UserProfile.jsx [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+                          viewBox="0 0 24 24"
+                          fill="none"
+                          stroke="currentColor"
+                          stroke-width="2"
+                          stroke-linecap="round"
+                          stroke-linejoin="round"
+                          class="feather feather-twitter me-2 icon-inline text-info"
+                        >
+                          <path d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"></path>
+                        </svg>
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:107]: why this ?

File: src/pages/user/UserProfile.jsx [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+                          xmlns="http://www.w3.org/2000/svg"
+                          width="24"
+                          height="24"
+                          viewBox="0 0 24 24"
+                          fill="none"
+                          stroke="currentColor"
+                          stroke-width="2"
+                          stroke-linecap="round"
+                          stroke-linejoin="round"
+                          class="feather feather-facebook me-2 icon-inline text-primary"
```
Comment (thakursachin05) [inline src/pages/user/UserProfile.jsx:153]: why this ?

**Review by thakursachin05 — CHANGES_REQUESTED** [review]
Please make these changes

File: src/component/questions/QuestionSeries.jsx (hunk 1 above)
Comment (Ashutosh741) [reply src/component/questions/QuestionSeries.jsx:33]: yeh to question component bana rhe , main to bas dekhne ke liye add kiya hoon

File: src/pages/user/UserProfile.jsx (hunk 4 above)
Comment (Ashutosh741) [reply src/pages/user/UserProfile.jsx:86]: these svg are logo , of social link, their quality is good that's why i have kept, as it is

Comment (Ashutosh741) [reply src/pages/user/UserProfile.jsx:86]: these svg are logo , of social link, their quality is good that's why i have kept, as it is

File: src/pages/user/UserProfile.jsx (hunk 3 above)
Comment (Ashutosh741) [reply src/pages/user/UserProfile.jsx:54]: state create kr diya hoon, ismein jab api fetch krunga, then use krunga

Comment (Ashutosh741) [reply src/pages/user/UserProfile.jsx:54]: state create kr diya hoon, ismein jab api fetch krunga, then use krunga

**Comment by Ashutosh741** [comment]
> Please make these changes

ok krta hoon

**Review by thakursachin05 — APPROVED** [review]

**Pull request merged**